code/script.js

```diff
@@ -1,9 +1,43 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
+
+console.log('CONNECTED')
+
 function toggle() {
   this.classList.toggle("active")
 }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+
+document.getElementById('form').onsubmit = (event) => {
+event.preventDefault()
+
+ const inputValue = document.getElementById('nameInput').value
+
+ document.getElementById('greeting').innerText= 'hello'
+}
+
+document.getElementById('form').onsubmit = event => {
+    event.preventDefault()
+  
+    // console.log('Form submittegit commit d')
+    // alert('Form submitted')
+    const inputValue = document.getElementById('nameInput').value
```

Comment:
You have declared a const inputValue on both line 13 and 23, you only need to the declare the inputValue once, then you can use it again if you declared it as a global variable (e.i. not within  function) :)